src/core/ui/components/primitives/chip.module.css

```diff
@@ -0,0 +1,14 @@
+.chip {
```

Comment:
What does chip mean?

Reply:
Its a kind of component, [popularized by material design](https://m3.material.io/components/chips/overview)
<img width="570" alt="image" src="https://user-images.githubusercontent.com/17853818/215224525-7d5c7d2b-c8ad-45d0-b386-abe8a5988dfd.png">

If you have a better name please let me know!